python-avd/pyavd/_eos_designs/schema/schema_fragments/connected_endpoints_keys.schema.yml

```diff
@@ -6,14 +6,24 @@
 # Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
 type: dict
 keys:
+  custom_connected_endpoints_keys:
+    $ref: eos_designs#/keys/connected_endpoints_keys
+    default: null
+    type: list
+    documentation_options:
+      table: connected-endpoints-keys
+    description: |-
+      Use this setting to specify values for `connected_endpoints_keys` in addition to the default ones.
```

Comment:
do not say "in addition" please use same wording as for custom_node_type_keys:


```
    description: |-
      Define Custom Node Type Keys, to specify the properties of each node type in the fabric.
      This allows for complete customization of the fabric layout and functionality.
      `custom_node_type_keys` should be defined in top level group_var for the fabric.
      These values will be combined with the defaults; custom node type keys named the same as a
      default node_type_key will replace the default.
```

Reply:
Fixed!